modules/nf-core/scimap/spatiallda/main.nf

```diff
@@ -0,0 +1,50 @@
+process SCIMAP_SPATIALLDA {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "docker.io/miguelib/scimap:0.0.3"
```

Comment:
I would suggest creating a repository with the Dockerfile, continuous integration and documentation about required and optional CLI parameters for clearer usage in MCMICRO (or other pipelines). Additionally, the build should be done to support both arm64 and amd64 architectures if possible.